chapel.html.markdown

```diff
@@ -1,11 +1,11 @@
 ---
-language: chapel
-filename: learnchapel.chpl
+language: Chapel
```

Comment:
This should be changed back to "chapel" as it will break the site more then likely.